polyply/src/load_library.py

```diff
@@ -101,12 +137,64 @@ def load_library(name, lib_names, extra_files):
     """
     force_field = vermouth.forcefield.ForceField(name)
     if lib_names and extra_files:
-        lib_files = _resolve_lib_paths(lib_names, DATA_PATH)
+        lib_files = _resolve_lib_paths(lib_names, DATA_PATH,
+                                       FORCE_FIELD_PARSERS.keys())
         all_files = lib_files + extra_files
     elif lib_names:
-        all_files = _resolve_lib_paths(lib_names, DATA_PATH)
+        all_files = _resolve_lib_paths(lib_names, DATA_PATH,
+                                       FORCE_FIELD_PARSERS.keys())
     elif extra_files:
         all_files = extra_files
 
     read_ff_from_files(all_files, force_field)
     return force_field
+
+
+def read_build_options_from_files(paths, topology):
+    """
+    read the input files for the build options and
+    molecule templates.
+
+    Parameters
+    ----------
+    paths: list[`os.path`]
+           List of valid file paths
+    topology: :class:`polyply.src.topology`
+
+    Returns
+    -------
+
+    """
+    for path in paths:
+        with open(path, 'r') as file_:
+            lines = file_.readlines()
+        read_build_file(lines, topology.molecules, topology)
+
+
+def load_build_options(topology, lib_names, build_file):
+    """
+    Load build file options and molecule templates into topology.
+
+    Parameters
+    ----------
+    topology: :class:`polyply.src.topology`
+    build_file: str
+        List of build files to parse
+    lib_names: list[str]
+        List of library names for which to load templates
+
+    Returns
+    -------
+
+    """
+    if lib_names and build_file:
+        lib_files = _resolve_lib_paths(lib_names, DATA_PATH, ['bld'])
+        all_files = lib_files + [build_file]
+    elif lib_names:
+        all_files = _resolve_lib_paths(lib_names, DATA_PATH, ['bld'])
+    elif build_file:
+        all_files = [build_file]
+    else:
+        return
+
+    read_build_options_from_files(all_files, topology)
```

Comment:
```suggestion
    all_files = _resolve_lib_paths(lib_names, DATA_PATH, ['bld'])
    if build_file:
         all_files += [build_file]
    
    read_build_options_from_files(all_files, topology)
```
But do you need this if? In what scenario do I call this function without a build_file?